src/shared/services/fields/fields.utils.ts

```diff
@@ -128,8 +128,8 @@ export class FieldsUtils {
     };
   }
 
-  public cloneObject(obj: any): any {
-    return Object.assign({}, obj);
+  public cloneObject(obj: any): CaseField {
```

Comment:
the `cloneObject` method was used to clone any type of object, you changed it to clone only CaseField. 
Rather than doing that, I think we should leave `cloneObject` as it was and make it public so it can be reused everywhere. 
And you can add a new public method `cloneCaseField`

Reply:
This method was only used for case fields and locally, I felt it to be changed for it.
I will change accordingly

Reply:
changed